### ISSUE TITLE
Email configuration from config/auth is not being used.

### ISSUE THREAD

**Issue by franciscocorrales** [Laravel-Backpack/Base#54]
Like the docs says, I set this in the _config/auth.php_ file:
```
'passwords' => [
        'users' => [
            'email' => 'backpack::auth.emails.password',
        ],
    ],
```

Then I edited it to use my own view, and set its content. The problem is that when I tried to send the Forgot Password email, I got a different content. So, I guess Backpack is not using this configuration (or at least not when the user requests for a reset password link). Can you check this?, am I doing something wrong ?

### More information:

Right now, it uses the views in /vendor/laravel/framework/src/illuminate/notifications/resources/views/

### This is my current solution:

Extend the PasswordBroker, so we can use our own notification class
```
class MyPasswordBroker extends PasswordBroker
{
    public function sendResetLink(array $credentials)
    {
...
        $token = $this->tokens->create($user);
        $user->notify(new ResetPasswordNotification($token));
...
    }
```
Extend Backpack's Reset Password Notification

```
class ResetPasswordNotification extends ResetPassword
{
...
new MailMessage())
   			->view('auth.emails.password',
	   				[
	   					'token' => $this->token,
	   					'moreData'	=> null
	   				]
   				);
```

### Settings:
Laravel 5.3.21
Backpack Base 0.7.3

**Comment by tabacitu** [comment]
Hmmm. Hi @franciscocorrales , 

Are you running on Laravel 5.2 or 5.3?

**Comment by franciscocorrales** [comment]
@tabacitu I updated the post.

**Comment by dereksdev** [comment]
Was this ever resolved? I am having the same issue in Laravel 5.4. Backpack appears to ignore this config value.

**Comment by lloy0076** [comment]
If this is still an issue, please reopen.
